### PR TITLE
Add dynamic minutes step in <TimePicker />

### PR DESCRIPTION
**Pull request by DanielEliraz**
Adding minutesStep prop in timePicker.
This prop set the number of minutes to be changed on arrow click.

- 📚 Change is documented
  - [X] Story
  - [X] API description
- 🔬 Change is tested
  - [X] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by DanielEliraz** [comment]
Hi @itzikkit, please tag Or also here.
And let's try to push it..

**Comment by itzikkit** [comment]
Added @orgol99, our PM

File: src/TimeInput/TimeInput.js [hunk 1]
```diff
@@ -45,6 +48,7 @@ export default class TimePicker extends Component {
     disableAmPm: false,
     disabled: false,
     dashesWhenDisabled: false,
+    minutesStep: 20,
```
Comment (EgoziE) [inline src/TimeInput/TimeInput.js]: Is it approved by our UX that the default step is 20 min?
I seems right to me that if I don't provide this prop (or sets it to 0) it gos back to 1 hour step as before.

Comment (DanielEliraz) [reply src/TimeInput/TimeInput.js]: It was 20 minutes before, only if the user selected the hour in the picker than it was 1 hour

Comment (DanielEliraz) [reply src/TimeInput/TimeInput.js]: `const timeUnit = this.state.lastFocusedTimeUnit || 'minutes';
const amount = timeUnit === 'hours' ? 1 : 20;`

Comment (EgoziE) [reply src/TimeInput/TimeInput.js]: there was not default value before

Comment (EgoziE) [reply src/TimeInput/TimeInput.js]: I'm talking about `defaultProps`

Comment (DanielEliraz) [reply src/TimeInput/TimeInput.js]: OK, but it was default (and only) in the code

Comment (EgoziE) [reply src/TimeInput/TimeInput.js]: I talked with our UX, this behaviour will change anyway when we rewrite this component in wsr8
So we'll merge it for now :)

**Comment by simast** [comment]
Let's not forget to add TypeScript typings for this new prop as well:
https://github.com/wix/wix-ui/blob/851eebff3b0b67cef3f7a6c218b28909d947c7a4/packages/wsr-types/src/components/TimeInput.d.ts#L3-L12

**Pull request merged**